frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -980,7 +980,7 @@ export const ClusterServiceVersionsDetailsPage: React.FC<ClusterServiceVersionsD
                   href: referenceForProvidedAPI(desc),
                   name: ['Details', 'YAML', 'Subscription', 'Events'].includes(desc.displayName)
```

Comment:
Not new, but we should probably ignore case on this. We probably don't want a 'Details' and 'details' tab. 
```suggestion
                  name: ['details', 'yaml', 'subscription', 'events'].includes(desc.displayName?.toLowerCase())
```

Reply:
@TheRealJon I see you point. However I think this need more discussion.

Reply:
@cyril-ui-developer Yeah, maybe something that could be a follow-on. We could also just say that it becomes the CRD author's responsibility to prevent naming collisions like this, but at the same time, it wouldn't be that hard for us to catch some easy edge cases here.